include/inplace_vector.hpp

```diff
@@ -271,8 +271,14 @@ class inplace_vector {
     constexpr const_iterator cend() const noexcept { return &m_data[0].ref() + m_size; }
     constexpr const_iterator begin() const noexcept { return cbegin(); }
     constexpr const_iterator end() const noexcept { return cend(); }
+    constexpr const_reverse_iterator crbegin() const noexcept { return const_reverse_iterator(end()); }
+    constexpr const_reverse_iterator crend() const noexcept { return const_reverse_iterator(begin()); }
+    constexpr const_reverse_iterator rbegin() const noexcept { return crbegin(); }
+    constexpr const_reverse_iterator rend() const noexcept { return crend(); }
```

Comment:
`rbegin`/`rend` should return `reverse_iterator`

Reply:
Sorry, I didn't see that these where the `const` versions. No problem with this.